src/page-modules/assistant/details/index.tsx

```diff
@@ -47,14 +48,7 @@ export function AssistantDetails({ tripPattern }: AssistantDetailsProps) {
         <DetailsHeader tripPattern={tripPattern} />
       </div>
       <div className={style.mapContainer}>
-        <Map
-          position={{
-            lon: tripPattern.legs[0].fromPlace.longitude,
-            lat: tripPattern.legs[0].fromPlace.latitude,
-          }}
-          initialZoom={13.5}
-          mapLegs={mapLegs}
-        />
+        <Map mapLegs={mapLegs} bounds={[bounds.sw, bounds.ne]} />
```

Comment:
Could this be baked in to the component instead where as it always show the entire trip? Or atleast something like `showAllLegsInView` as a property and move invocation of `getMapBounds(mapLegs)` encapsulated by `Map` component?

Reply:
Updated to reflect this now. When `mapLegs` is defined we want to show the whole trip, which makes your suggestion make much sense 👍